.github/workflows/deploy-auth.yml

```
@@ -0,0 +1,59 @@
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: auth
+
+    steps:
+      - name: 📦 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 🛠️ Set up JDK
+        uses: actions/setup-java@v3
```

actionlint (1.7.4)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)